### PR TITLE
[Feature](bangc-ops): update binary-ops version to 1.15.2.

### PR DESCRIPTION
**Pull request by Unireverse**
Thanks for your contribution and we appreciate it a lot. 

## 1. Motivation

Update binary ops version to 1.15.2.

## 2. Modification

bangc-ops/kernels/kernel_wrapper/lib/libextops.a

## 3. Test Report

None

### CODE REVIEW

File: bangc-ops/mlu_op.h [hunk 1]
```diff
@@ -1794,6 +1794,209 @@ mluOpAbs(mluOpHandle_t handle,
...
+ * The number of tensors in array inputs[].
+ * @param[in] output_desc
+ * The descriptor of the output tensor, For detailed information, see
+ * ::mluOpTensorDescriptor_t.
+ * @param[out] workspace_size
+ * Host pointer to the returned size of the extra workspace in bytes that is
+ * used in add_n operation.
+ *
+ * @par Return
+ * - ::MLUOP_STATUS_SUCCESS, ::MLUOP_STATUS_BAD_PARAM,
```
Comment (WangYuan-cn) [inline bangc-ops/mlu_op.h:1818]:
```suggestion
 * - ::MLUOP_STATUS_SUCCESS, ::MLUOP_STATUS_BAD_PARAM
```

File: bangc-ops/mlu_op.h [hunk 2]
```diff
@@ -1794,6 +1794,209 @@ mluOpAbs(mluOpHandle_t handle,
...
+ * @param[in] input_num
+ * Number of tensors in array inputs[].
+ * @param[in] output_desc
+ * Descriptor of the output tensor. For detailed information, see
+ * ::mluOpTensorDescriptor_t.
+ * @param[out] output
+ * Device pointer to the MLU memory that stores the output tensor.
+ * @param[in] workspace
+ * Device pointer to the MLU memory that is used as an extra workspace for this
+ * operation. For more information about workspace, see "Cambricon MLUOP User Guide".
```
Comment (WangYuan-cn) [inline bangc-ops/mlu_op.h:1865]: BANGC

File: bangc-ops/mlu_op.h [hunk 3]
```diff
@@ -1794,6 +1794,209 @@ mluOpAbs(mluOpHandle_t handle,
...
+ * - Before calling this function to perform AddN operation, you need to get the size
+ *   of workspace by the ::mluOpGetAddNWorkspaceSize function.
+ *
+ * @par Data Type
+ * - This function supports the following data types for input and output tensors.
+ * Note that the data type of output should be same with input.
+ *   - input tensor: float, half, int32, int16, int8, uint8.
+ *   - output tensor: float, half, int32, int16, int8, uint8.
+
+ *
```
Comment (WangYuan-cn) [inline bangc-ops/mlu_op.h:1883]: 多了行空格，1886 少了一行空格

File: bangc-ops/mlu_op.h [hunk 4]
```diff
@@ -1794,6 +1794,209 @@ mluOpAbs(mluOpHandle_t handle,
...
+ * workspace to optimize the ::mluOpAddN_v2 operation.
+ *
+ * @param[in] handle
+ * Handle to an MLUOP context that is used to manage MLU devices and queues in the add_n
+ * operation. For detailed information, see ::mluOpHandle_t.
+ * @param[in] input_descs[]
+ * Array of descriptors for all input tensors. For detailed information,
+ * see ::mluOpTensorDescriptor_t.
+ * @param[in] input_num
+ * The number of tensors in array inputs[].
```
Comment (GraceGuanH) [inline bangc-ops/mlu_op.h:1809]: []这个需要加吗？

File: bangc-ops/mlu_op.h [hunk 5]
```diff
@@ -1794,6 +1794,209 @@ mluOpAbs(mluOpHandle_t handle,
...
+ * @param[in] output_desc
+ * The descriptor of the output tensor, For detailed information, see
```
Comment (GraceGuanH) [inline bangc-ops/mlu_op.h:1811]: tensor后的逗号改为句号

File: bangc-ops/mlu_op.h [hunk 6]
```diff
@@ -1794,6 +1794,209 @@ mluOpAbs(mluOpHandle_t handle,
...
+mluOpStatus_t MLUOP_WIN_API mluOpGetAddNWorkspaceSize(mluOpHandle_t handle,
+                                                      const mluOpTensorDescriptor_t input_descs[],
+                                                      const uint32_t input_num,
+                                                      const mluOpTensorDescriptor_t output_desc,
+                                                      size_t *workspace_size);
+// Group:AddN
+/*!
+ * @brief Computes the sum of input tensors.
+ *
+ * AddN operation is wildly used in artificial intelligence as a kind of basic mathematical
```
Comment (GraceGuanH) [inline bangc-ops/mlu_op.h:1839]: 这里是wildly还是widely    artificial intelligence 》AI

File: bangc-ops/mlu_op.h [hunk 7]
```diff
@@ -1794,6 +1794,209 @@ mluOpAbs(mluOpHandle_t handle,
...
+ * operations. Also, this operation is supported in almost all common frameworks, like
+ * PyTorch and TensorFlow.
+ * Compared with ::mluOpAddN, this function supports multidirectional broadcasting of input tensors.
+ *
+ * This function may need extra MLU memory as the workspace to support multidirectional broadcasting.
+ * You can get the size of the workspace \b workspace_size with the ::mluOpGetAddNWorkspaceSize
+ * function.
+ *
+ * @param[in] handle
+ * Handle to an MLUOP context that is used to manage MLU devices and queues in add_n
```
Comment (GraceGuanH) [inline bangc-ops/mlu_op.h:1849]: add_n  咱们有这个算子名称吗？

File: bangc-ops/mlu_op.h [hunk 8]
```diff
@@ -1794,6 +1794,209 @@ mluOpAbs(mluOpHandle_t handle,
...
+ * operation. For detailed information, see ::mluOpHandle_t.
+ * @param[in] input_descs[]
+ * Array of descriptors for all input tensors. For detailed information,
+ * see ::mluOpTensorDescriptor_t.
+ * @param[in] inputs[]
+ * Array of device pointers to the MLU memory for the input tensors.
+ * @param[in] input_num
+ * Number of tensors in array inputs[].
```
Comment (GraceGuanH) [inline bangc-ops/mlu_op.h:1857]: 确认下这三处的[] 是否需要

File: bangc-ops/mlu_op.h (hunk 2 above)
Comment (GraceGuanH) [reply bangc-ops/mlu_op.h:1865]: Cambricon BANGC OPS User Guide

File: bangc-ops/mlu_op.h [hunk 9]
```diff
@@ -1794,6 +1794,209 @@ mluOpAbs(mluOpHandle_t handle,
...
+ * operation. For more information about workspace, see "Cambricon MLUOP User Guide".
+ * @param[in] workspace_size
+ * Size of the extra workspace in bytes that needs to be used in this
+ * operation. You can get the size of workspace with the ::mluOpGetAddNWorkspaceSize
+ * function.
+ * @par Return
+ * - ::MLUOP_STATUS_SUCCESS, ::MLUOP_STATUS_BAD_PARAM
+ *
+ * @par API Dependency
+ * - Before calling this function to perform AddN operation, you need to get the size
```
Comment (GraceGuanH) [inline bangc-ops/mlu_op.h:1874]: AddN 写完整算子的名称

File: bangc-ops/mlu_op.h [hunk 10]
```diff
@@ -1794,6 +1794,209 @@ mluOpAbs(mluOpHandle_t handle,
...
+ *   of workspace by the ::mluOpGetAddNWorkspaceSize function.
+ *
+ * @par Data Type
+ * - This function supports the following data types for input and output tensors.
```
Comment (GraceGuanH) [inline bangc-ops/mlu_op.h:1878]: 去掉 -

File: bangc-ops/mlu_op.h [hunk 11]
```diff
@@ -1794,6 +1794,209 @@ mluOpAbs(mluOpHandle_t handle,
...
+ * Note that the data type of output should be same with input.
```
Comment (GraceGuanH) [inline bangc-ops/mlu_op.h:1879]: the same with that of the input.

File: bangc-ops/mlu_op.h [hunk 12]
```diff
@@ -1794,6 +1794,209 @@ mluOpAbs(mluOpHandle_t handle,
...
+ *   - input tensor: float, half, int32, int16, int8, uint8.
+ *   - output tensor: float, half, int32, int16, int8, uint8.
```
Comment (GraceGuanH) [inline bangc-ops/mlu_op.h:1881]: 这两处去掉句尾的句号

File: bangc-ops/mlu_op.h [hunk 13]
```diff
@@ -1794,6 +1794,209 @@ mluOpAbs(mluOpHandle_t handle,
...
+ *
+ * @par Deprecated
+ * - ::mluOpAddN is deprecated and will be removed in the future release. It is recommended to use
+ *   ::mluOpAddN_v2 instead.
+ *
+ * @param[in] handle
+ * Handle to an MLUOP context that is used to manage MLU devices and queues in this
+ * operation. For detailed information, see ::mluOpHandle_t.
+ * @param[in] input_descs[]
+ * Array of descriptor for the all input tensors. For detailed information,
```
Comment (GraceGuanH) [inline bangc-ops/mlu_op.h:1934]: for all the input tensors

File: bangc-ops/mlu_op.h [hunk 14]
```diff
@@ -1794,6 +1794,209 @@ mluOpAbs(mluOpHandle_t handle,
...
+ * see ::mluOpTensorDescriptor_t.
+ * @param[in] inputs[]
+ * Array of device pointers to the MLU memory for the all input tensors.
+ * @param[in] input_num
+ * Number of tensors in array inputs[].
+ * @param[in] output_desc
+ * Descriptor of the output tensor, For detailed information, see
```
Comment (GraceGuanH) [inline bangc-ops/mlu_op.h:1941]: The descriptor of the output tensor.

File: bangc-ops/mlu_op.h [hunk 15]
```diff
@@ -1794,6 +1794,209 @@ mluOpAbs(mluOpHandle_t handle,
...
+ * @param[out] output
+ * Device pointer to the MLU memory that stores the output tensor.
+ * @par Return
+ * - ::MLUOP_STATUS_SUCCESS, ::MLUOP_STATUS_BAD_PARAM,
+ *   ::MLUOP_STATUS_ARCH_MISMATCH, ::MLUOP_STATUS_NOT_SUPPORTED
+ *
+ * @par Data Type
+ * - This function supports the following data types for input and output tensors.
+ *   - input tensor: float, half.
+ *   - output tensor: float, half.
```
Comment (GraceGuanH) [inline bangc-ops/mlu_op.h:1952]: 两句中句尾的句号去掉

**Review by GraceGuanH — COMMENTED** [review]
Please check the comments.

**Review by WangYuan-cn — APPROVED** [review]

File: bangc-ops/mlu_op.h (hunk 8 above)
Comment (Unireverse) [reply bangc-ops/mlu_op.h:1857]: 需要的

**Pull request merged**